### ISSUE TITLE
Collecting VR demonstrations. 

### ISSUE THREAD

**Issue by nikwl** [ikalevatykh/mime-release#1]
Thanks for releasing your code! I'm working with pybullet, and I was curious as to what degree your project implements VR teleportation. There seem to be several utility scripts that would indicate you support using VR to collect expert demonstrations but I have yet to see any way to actually go about collecting VR data. 

Specifically, though the Scene superclass has a docstring mentioning VR support, it doesn't seem to support connecting to external physics servers, and the structure of the project is such that the urdf files need to be loaded on the local machine so connecting to an external server seems impossible.

TLDR; does your repo support VR teleportation (and if so, how exactly do you go about collecting VR data)?

Thanks!
